src/ripple/core/ConfigSections.h

```diff
@@ -55,7 +55,6 @@ struct ConfigSection
 #define SECTION_PATH_SEARCH_MAX         "path_search_max"
 #define SECTION_PEER_PRIVATE            "peer_private"
 #define SECTION_PEERS_MAX               "peers_max"
-#define SECTION_RPC_ADMIN_ALLOW         "rpc_admin_allow"
```

Comment:
We've not warned in the past when deprecating config sections. Its probably a good idea to warn as you suggested (separate PR). I did remove rpc_admin_allow from the example config file, added new docs on the admin setting.